src/constants/routes.ts

```diff
@@ -221,8 +221,14 @@ export const getProjectPatchesRoute = (projectId: string) =>
 
 export const getProjectSettingsRoute = (
   projectId: string,
-  tab: ProjectSettingsTabRoutes
-) => `${paths.project}/${projectId}/${PageNames.Settings}/${tab}`;
+  tab?: ProjectSettingsTabRoutes
+) => {
+  if (!tab) {
+    return `${paths.project}/${projectId}/${PageNames.Settings}/${ProjectSettingsTabRoutes.General}`;
+  }
```

Comment:
I'd rather not hardcode directing to the general tab here since that is defined elsewhere as the default page and is theoretically subject to change. If no tab is defined, can we direct to `${paths.project}/${projectId}/${PageNames.Settings}`? I think then the user will be redirected to the General page by our router.

Reply:
good point, fixed